aten/src/ATen/native/cpu/TensorCompareKernel.cpp

```diff
@@ -161,10 +161,76 @@ static void where_kernel_impl(TensorIterator &iter, ScalarType condition_type) {
   });
 }
 
+static void isposinf_kernel_for_bool_impl(TensorIterator& iter) {
+  if (c10::isIntegralType(iter.input_dtype(), /*include_bool=*/true)) {
+    AT_DISPATCH_INTEGRAL_TYPES_AND(at::ScalarType::Bool, iter.input_dtype(), "isposinf_cpu", [&]() {
```

Comment:
This dispatch would also be covered by the above trick.

Reply:
addressed